src/backend/marsha/core/api/video.py

```diff
@@ -322,6 +326,9 @@ def start_live(self, request, pk=None):
         start_live_channel(video.get_medialive_channel().get("id"))
 
         video.live_state = defaults.STARTING
+        video.upload_state = defaults.PENDING
+        video.recording_slices = []
```

Comment:
So at every start the `recording_slices` is reset ? I don't know if it's really what we want.